exercises/beer-song/inputdata.txt

```diff
@@ -0,0 +1,7 @@
+99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n
+44 bottles of beer on the wall, 44 bottles of beer.\nTake one down and pass it around, 43 bottles of beer on the wall.\n
```

Comment:
I notice that there is a new line literal (though not literal, as I can see it) at the end of each of these lines.

This probably should be a plain old text file, formatted as it needs to be.  This will make the "\n" characters go away.